rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/license/Sw360LicenseService.java

```diff
@@ -113,11 +113,9 @@ public void deleteAllLicenseInfo(User user) throws TException {
     public License createLicense(License license, User sw360User) throws TException {
         LicenseService.Iface sw360LicenseClient = getThriftLicenseClient();
         license.setId(license.getShortname());
-        List<License> licenses = sw360LicenseClient.addLicenses(Collections.singletonList(license), sw360User);
-        for (License newLicense : licenses) {
-            if (license.getFullname().equals(newLicense.getFullname())) {
-                return newLicense;
-            }
+        RequestStatus requestStatus = sw360LicenseClient.updateLicense(license, sw360User, sw360User);
```

Comment:
why directly updateLicense method instead of addLicenses while creating a new license?

Reply:
Reverted the changes